components/admin_console/permission_schemes_settings/guest_permissions_tree/guest_permissions_tree.test.tsx

```diff
@@ -92,22 +89,6 @@ describe('components/admin_console/permission_schemes_settings/permission_tree',
         expect(onToggle).toBeCalledWith('test', ['test_permission', 'test_permission2']);
     });
 
-    test('should hide disabbled integration options', () => {
```

Comment:
`config` and related test is removed as there is no `config` prop in the component